<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
     <property name="geometry">
         <rect>
             <x>0</x>
             <y>0</y>
             <width>740</width>
             <height>400</height>
         </rect>
     </property>
     <property name="windowTitle">
         <string>Simple Audio Player</string>
     </property>
     <widget class="QWidget" name="centralwidget">
         <widget class="QSplitter" name="hsplitter">
             <property name="orientation">
                 <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QTreeView" name="treeView">
                 <property name="font">
                     <font>
                         <pointsize>12</pointsize>
                     </font>
                 </property>
             </widget>
         </widget>
         <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
                 <rect>
                     <x>0</x>
                     <y>300</y>
                     <width>740</width>
                     <height>100</height>
                 </rect>
             </property>
             <layout class="QHBoxLayout" name="controlsLayout">
                 <item>
                     <widget class="QPushButton" name="fileButton">
                         <property name="toolTip">
                             <string>Datei(en) auswählen</string>
                         </property>
                         <property name="text">
                             <string/>
                         </property>
                         <property name="iconSize">
                             <size>
                                 <width>32</width>
                                 <height>32</height>
                             </size>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <widget class="QPushButton" name="folderButton">
                         <property name="toolTip">
                             <string>Verzeichnis auswählen</string>
                         </property>
                         <property name="text">
                             <string/>
                         </property>
                         <property name="iconSize">
                             <size>
                                 <width>32</width>
                                 <height>32</height>
                             </size>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <widget class="QPushButton" name="radioButton">
                         <property name="toolTip">
                             <string>Internet Radio</string>
                         </property>
                         <property name="toolTipDuration">
                             <number>-2</number>
                         </property>
                         <property name="text">
                             <string/>
                         </property>
                         <property name="iconSize">
                             <size>
                                 <width>32</width>
                                 <height>32</height>
                             </size>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <widget class="QPushButton" name="playButton">
                         <property name="toolTip">
                             <string>Abspielen / Pause</string>
                         </property>
                         <property name="text">
                             <string/>
                         </property>
                         <property name="iconSize">
                             <size>
                                 <width>32</width>
                                 <height>32</height>
                             </size>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <widget class="QSlider" name="volumeSlider">
                         <property name="maximumSize">
                             <size>
                                 <width>170</width>
                                 <height>21</height>
                             </size>
                         </property>
                         <property name="baseSize">
                             <size>
                                 <width>170</width>
                                 <height>21</height>
                             </size>
                         </property>
                         <property name="toolTip">
                             <string>Lautstärke </string>
                         </property>
                         <property name="value">
                             <number>30</number>
                         </property>
                         <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <widget class="QPushButton" name="speakerButton">
                         <property name="text">
                             <string/>
                         </property>
                         <property name="iconSize">
                             <size>
                                 <width>32</width>
                                 <height>32</height>
                             </size>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <widget class="QLabel" name="posLabel">
                         <property name="minimumSize">
                             <size>
                                 <width>36</width>
                                 <height>21</height>
                             </size>
                         </property>
                         <property name="text">
                             <string/>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <widget class="QProgressBar" name="progressBar">
                         <property name="value">
                             <number>0</number>
                         </property>
                         <property name="textVisible">
                             <bool>false</bool>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <widget class="QLabel" name="lenLabel">
                         <property name="minimumSize">
                             <size>
                                 <width>36</width>
                                 <height>21</height>
                             </size>
                         </property>
                         <property name="text">
                             <string/>
                         </property>
                     </widget>
                 </item>
             </layout>
         </widget>
        <widget class="QStatusBar" name="statusbar"/>
     </widget>
  </widget>
</ui>
